vizro-ai/tests/integration/test_dashboard.py

```diff
@@ -21,10 +22,15 @@ def setup_test_environment():
         chromedriver_autoinstaller.install()
 
 
+@pytest.mark.parametrize(
+    "model_name",
+    ["gpt-3.5-turbo"],
+    ids=["gpt-3.5"],
+)
```

Comment:
why I seems to receive 2 sets of test result in the output file? Is it because the "append" mode of CSV? In that case, does it mean it will keep appending infinitely on the runner machine?
<img width="284" alt="Screenshot 2024-08-27 at 17 22 27" src="https://github.com/user-attachments/assets/c438c482-0cbd-4ada-b0b0-71ec993e2a0a">

Reply:
That is because on the CI we have two runs in one job. One for the vizro from pypi and one for the vizro from the branch.

Reply:
Could we label the different reports?